### PR TITLE
Picard 2.3.1 update

### PR DESCRIPTION
**Pull request by phw**
# Summary
Changes for Picard 2.3.1

- [x] Added _releaselcountries to tag documentation

* This is a…
    * [ ] Bug fix
    * [x] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): [PW-XXX](https://tickets.metabrainz.org/browse/PW-XXX)

# Solution

# Action

### CODE REVIEW

File: website/frontend/templates/docs/scripting.html [hunk 1]
```diff
@@ -315,6 +315,24 @@ <h3 id="map_name_code_separator">$map(name,code,separator="; ")</h3>
           </li>
         </ul>
 
+        <h3 id="sortmulti_name_separator">$sortmulti(name,separator="; ")</h3>
+
+        <ul>
+          <li>Returns a copy of the multi-value tag
+            <code>name</code> with the elements sorted in ascending order.
```
Comment (zas) [inline website/frontend/templates/docs/scripting.html]: Shouldn't `separator` be documented?

File: website/frontend/templates/docs/tags.html [hunk 2]
```diff
@@ -325,6 +325,13 @@ <h1 id="basic_variables">Basic Variables</h1>
            >Release Language</a> as per <a rel="nofollow" class="external text" href="http://www.sil.org/iso639-3">ISO 639-3</a> (since
             Picard 0.10)</td>
         </tr>
+        <tr>
+          <td>_releaselcountries</td>
```
Comment (zas) [inline website/frontend/templates/docs/tags.html]: Is `_releaselcountries` correct??

Comment (phw) [reply website/frontend/templates/docs/tags.html]: Yes

Comment (phw) [reply website/frontend/templates/docs/tags.html]: ah, you mean there is a typo in _releasecountries. Thought you were asking about the underscore.

**Pull request merged**